cg/apps/demultiplex/sample_sheet/base.py

```diff
@@ -48,26 +17,10 @@ def __init__(self, message: str):
 
 
 class SampleSheetError(ModelError):
-    """Raised when something is wrong with the orderform."""
+    """Raised when something is wrong with the order form."""
 
 
-class BaseSample(BaseModel):
```

Comment:
Not needed?

Reply:
The original code in cgmodels seemed to handle NovaSeq and HiSeq samples. As in cg we only handle NovaSeq, I thought the distinction was unnecessary, but I will double check 😉